core/src/retransmit_stage.rs

```diff
@@ -165,8 +182,7 @@ fn maybe_reset_shreds_received_cache(
     packet_hasher: &mut PacketHasher,
```

Comment:
Why do we need to reset the shreds received cache at all?

Reply:
Clearing the cache will clear the duplicate versions. It seems safer to keep the existing behavior.